src/frontend/components/SøknadsSteg/OmBarnet/useOmBarnet.tsx

```diff
@@ -202,41 +191,31 @@ export const useOmBarnet = (
     });
 
     /*---UTENLANDSOPPHOLD---*/
-    const barnRegistrerteUtenlandsperioder = useFelt<IUtenlandsperiode[]>({
-        feltId: UtenlandsoppholdSpørsmålId.utenlandsopphold,
-        verdi: gjeldendeBarn.utenlandsperioder,
-        valideringsfunksjon: felt => {
+    const {
+        fjernPeriode: fjernUtenlandsperiodeBarn,
+        leggTilPeriode: leggTilUtenlandsperiodeBarn,
+        registrertePerioder: barnRegistrerteUtenlandsperioder,
+    } = usePerioder<IUtenlandsperiode>(
+        gjeldendeBarn.andreForelder?.utenlandsperioder ?? [],
+        {},
```

Comment:
Ønsker egentlig å refaktorere denne i en egen PR, til å ta inn et objekt for lesebarhetens skyld